draco/analysis/mapmaker.py

```diff
@@ -194,7 +182,7 @@ class MaximumLikelihoodMapMaker(BaseMapMaker):
     The dirty map is produced by generating a set of :math:`a_{lm}` coefficients
     using
 
-    .. math:: \hat{\mathbf{a}} = \left( \mathbf{N}^{-1/2 }\mathbf{B} \right)^+ \mathbf{N}^{-1/2} \mathbf{v}
+    .. math:: \hat{\mathbf{a}} = ( \mathbf{N}^{-1/2 }\mathbf{B} ) ^+ \mathbf{N}^{-1/2} \mathbf{v}
```

Comment:
@jrs65 I couldn't find out what `\left` and `\right` did, but they made this break with an error like "Expected an empty line, unexpected unindent" or similar

Reply:
oh, I see: it ensures the brackets are big enough to encapsulate the entire block inside. Let's see what comes out of this. Apart from the warning, it definitely produces garbage *with* the `\right`,`\left`

Reply:
It could well be that the `\` are getting interpreted as escape codes in the string. Try changing the string to be a raw string, i.e. start it with `r"""`

Reply:
thanks, that worked! (and the brackets really are a tiny bit bigger now :facepalm: )